src/components/CloseButton/CloseButton.module.css

```diff
@@ -0,0 +1,48 @@
+.close-button {
+  width: var(--eds-size-6);
+  height: var(--eds-size-6);
+  border-radius: 50%;
+
+  svg {
+    /* TODO(Icon): use size prop on Icon after that component is done */
+    height: var(--eds-size-4);
+    width: var(--eds-size-4);
+    /* TODO(Icon): use color prop on Icon after that component is done */
+    fill: var(--close-button-icon-color);
```

Comment:
I also wonder BM's thoughts on duplication vs. clarity / themability. Looking at their banner component, it seems they would do something like:
```
svg {
  // parent selector
  .close-button--variant-brand & {
     fill: ...
  }
  // and so on
```

instead of pulling out a `--close-button-icon-color` variable

Reply:
In this case, I expect this to be a short-term solution because our `Icon` component has internal color support, but it's good to have the conversation around whether we prefer duplication even if css variables would make it shorter and cleaner.

Reply:
BM avoids using HTML elements as selectors. Use a class instead of `svg` here.

Reply:
Actually now that I looked at this in context, we're modifying the svg that is rendered by the IconComponent, which in turn is being rendered as the text prop of the ButtonComponent. This feels like it's going against the Single Responsibility Principle to me.

Reply:
@csnizik as the TODO comments indicate, this is temporary styling that will be replaced when the `Icon` component can support the `color` and `size` props and therefore apply this styling itself. On `main` we're changing the text color on the button, which the svg inherits. https://github.com/chanzuckerberg/edu-design-system/blob/main/packages/components/src/CloseButton/CloseButton.module.css

That being said, I'll talk to Sean about whether we can use the standard icon button styles for this. I personally like this button's styling but I also want it to be consistent; we don't use this visual pattern anywhere else, which is why we have to do it manually instead of relying on the button component.